app/HMS/User/Permissions/RoleManager.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace HMS\User\Permissions;
+
+use HMS\Repositories\RoleRepository;
+use LaravelDoctrine\ORM\Facades\EntityManager;
+use LaravelDoctrine\ACL\Permissions\Permission;
+
+class RoleManager
+{
+    private $roleRepository;
+    private $permissionRepository;
+
+    /**
+     * Create a new RoleManager instance.
+     *
+     * @param HMS\Repositories\RoleRepository $roleRepository An instance of a role repository
+     */
+    public function __construct(RoleRepository $roleRepository)
+    {
+        $this->roleRepository = $roleRepository;
+        $this->permissionRepository = EntityManager::getRepository(Permission::class);
```

Comment:
Okay. In that case inject the entity manager and use that to create the repository is ready of the facade?